css/css.md

```diff
@@ -51,11 +52,11 @@ About 4.5 hours
 
 ### Lesson
 
-[CSS (video walkthrough of slides)](https://drive.google.com/file/d/1IyAozbB3BAuFXdAZH1tu0kr-eL3El0Cn/view?usp=sharing)
+[CSS (video walkthrough of slides)](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
```

Comment:
this is not a video.  please change it back.

Reply:
Done !

Reply:
This is still not a video, sorry. Always double check your links!